### ISSUE TITLE
bot

### ISSUE THREAD

**Issue by Brasil2021** [binary-com/binary-bot#3247]
hello, could you help me, i'm from brazil, the bots available here are to break the bank ..
the bot I have and even or odd, I use it in the odd,
could you help me and add to this bot the function of after 3 them he wait 4 to 8 seconds for new entry, always in odd.
a section where I can choose how many seconds I want ..
since already thank you, I am studying these bots, thank you

**Comment by Brasil2021** [comment]
this is the bot

[binary-bot par ou impra junior bom .zip](https://github.com/binary-com/binary-bot/files/6468928/binary-bot.par.ou.impra.junior.bom.zip)

**Comment by fperiin** [comment]
Do you want it to wait 4 to 8 seconds after 3 entries and make a new entry or after 3 consecutive losses?
If you prefer you can answer me in Portuguese, I am also from Brazil!

**Comment by fperiin** [comment]
> Do you want it to wait 4 to 8 seconds after 3 entries and make a new entry or after 3 consecutive losses?
> If you prefer you can answer me in Portuguese, I am also from Brazil!

I will anticipate and present a simple solution, here you will make it run in the range of 4 to 8 after each entry.
If you want, just use the same logic but within the gain or lose

[https://prnt.sc/12v96qf](https://prnt.sc/12v96qf)

**Comment by Brasil2021** [comment]
que bom que vc e do brasil amigo,sou de belo horizonte, queria que ele parasse em torno de 4 a 8 segundos depois de 3 loss consecutivo,pois ele vai bem e depois de um tempo ele da uns 8 a 10 loss direto com esses segundos evitaria essa seguencia tão grande ...não sei muita coisa mais estou estudando,comprei um curso soque ele e so para quebrar banca kkk,,ai fiquei curioso e estou aprendendo,,,se poder me ajudar com esse bot amigo..te agradeço

**Comment by Brasil2021** [comment]
se conseguir me mandar ele no formato que ele pare so na perda te agradeço,pois nen sei fazer kkkkk que ai ja coloco ele para testar

**Comment by fperiin** [comment]
Testa aí, acredito que ficou do jeito q vc queria


[3loss = 4-8 segundos.zip](https://github.com/binary-com/binary-bot/files/6474196/3loss.4-8.segundos.zip)

**Comment by Brasil2021** [comment]
correto amigo,vou testar ele agora ,,,obriga de verdade ,,,olha me desculpe a pergunta mais me interresso muito por essa area, tenho uma estrategia que queria aplicar em um bot,,,não sei se seria o caso mais se poder me chamar no zap, aqui 031994768346...ou posso falar por aqui mesmo? mais desde ja agradeço
pela sua ajuda vou testar o bot e te falo como ficou amigo Fperiin ...sou junior

**Comment by Xbrmachine** [comment]
Possuo o BOT mais perfeito já criado para a corretora binary e estou criando um grupo de investimento coletivo onde a partir de 10 dólares será possivel obter bons lucros.

Para saber mais acesse nosso telegram:
https://t.me/botbr

**Comment by masterkall** [comment]
@Brasil2021 tb so de BH Chama Whatss ai (31)99333-0407
